app/services/user_alerts/alert_user_about_account_verified.rb

```diff
@@ -1,12 +1,11 @@
 module UserAlerts
   class AlertUserAboutAccountVerified
-    def self.call(user:, date_time:, sp_name:, disavowal_token:)
```

Comment:
Build failure caught it, but the specs need to be updated.

Reply:
oh right, fixed in bdfdc10126d15d8476932797273a141e7e577165